app/views/course/achievements/_form.html.slim

```diff
@@ -7,29 +7,7 @@
 
   = f.input :draft
 
-  - if @achievement.try(:persisted?)
```

Comment:
to the `app/views/course/condition/_conditions.html.slim` [partial](https://github.com/Coursemology/coursemology2/pull/685/files#diff-c7c8bc68e2f0743a235559f2b9a11fbeR1) so can reuse in the assessment view